### ISSUE TITLE
Bold default TextInput font

### ISSUE THREAD

**Issue by mrkev** [trueinteractions/tint2#179]
I'm trying to bold the default font of a read-only TextInput using

    text.font.bold = true;

this however throws an AssertionError:

    AssertionError: A family name was not passed in for the font.

I can't even get the default system font. If I do

    console.log(text.font);

I get the same AssertionError. How can I make a TextInput with the default system font, but bolded?

**Comment by trevorlinton** [comment]
@mrkev Ah, good point.  By default it doesn't request the font from the system if one has not yet been assigned. 

A workaround is to assign your own font: 

```javascript
var font = new Font('Arial', 13);
font.bold
text.font = font;
```

I'll take a look at what we can do to fix this in the next release.

**Comment by mrkev** [comment]
Ah ok, thanks!

**Comment by trevorlinton** [comment]
Thanks for filing this issue, we really do appreciate you reporting this.
The workaround is no longer necessary, this will be released with 2.2.5 due next Monday (1/4/2016). 
Feel free to reopen if you continue to have issues.

**Comment by mrkev** [comment]
Perfect, thanks!
